index.css

```diff
@@ -0,0 +1,102 @@
+body {
+  display: block;
+  margin: 0;
+}
+
+.font {
+  font-family: 'Poppins', sans-serif;
+  margin: 4px;
+}
+
+.first-section {
+  background-image: url('./images/header-shapes-mobile.jpg');
+  height: 95vh;
+  width: 55vh;
+  background-position: 0 49px;
+  background-repeat: no-repeat;
+  margin: 0;
+  display: block;
+}
+
+.first-container {
+  height: 100%;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin: 11px;
```

Comment:
- [ ] This margin is adding extra overflow-x to your header and to your body layout. You can instead specify a vertical margin (eg: `margin: 0 11px;`) that was your intention.